src/dolphin/manager.ts

```diff
@@ -79,17 +79,22 @@ export class DolphinManager {
   }
 
   public async launchNetplayDolphin() {
-    if (this.netplayDolphinInstance) {
-      throw new Error("Netplay dolphin is already open!");
-    }
-
     await this._updateDolphinSettings(DolphinLaunchType.NETPLAY);
 
     const netplayInstallation = this.getInstallation(DolphinLaunchType.NETPLAY);
     const dolphinPath = await netplayInstallation.findDolphinExecutable();
     log.info(`Launching dolphin at path: ${dolphinPath}`);
     const launchMeleeOnPlay = this.settingsManager.get().settings.launchMeleeOnPlay;
```

Comment:
```suggestion
    const launchMeleeOnPlay = this.settingsManager.get().settings.launchMeleeOnPlay || 
this.netplayDolphinInstance !== null;
```
I think this achieves what you want without needing to change `meleeIsoPath` later on yeah?